dhis-2/dhis-api/src/main/java/org/hisp/dhis/analytics/common/dimension/DimensionParam.java

```diff
@@ -49,9 +49,11 @@
  */
 @Data
 @Builder( access = PRIVATE )
-@AllArgsConstructor( access = PRIVATE )
+@RequiredArgsConstructor( access = PRIVATE )
 public class DimensionParam
```

Comment:
Would be nice to have some Javadoc here.
It could explain why to we need this class and how it fits in the flow.